src/main/java/org/broadinstitute/hellbender/tools/exome/DecomposeSingularValues.java

```diff
@@ -84,7 +84,7 @@ protected void runPipeline(final JavaSparkContext ctx) {
     private void writeMatrix(final RealMatrix v, final File outputFilename) throws IOException {
         final List<String []> textTable = new ArrayList<>();
         for (int i = 0; i < v.getRowDimension(); i ++){
-            textTable.add(Arrays.stream(v.getRow(i)).mapToObj(Double :: toString).toArray(String[]::new));
+            textTable.add(Arrays.stream(v.getRow(i)).mapToObj(Double::toString).toArray(String[]::new));
```

Comment:
like Trinity or Morpheus?

On Tue, Jul 12, 2016 at 1:56 PM, David Benjamin <notifications@github.com>
wrote:

> In
> src/main/java/org/broadinstitute/hellbender/tools/exome/DecomposeSingularValues.java
> <https://github.com/broadinstitute/gatk-protected/pull/609#discussion_r70487732>
> :
>
> > @@ -84,7 +84,7 @@ protected void runPipeline(final JavaSparkContext ctx) {
> >      private void writeMatrix(final RealMatrix v, final File outputFilename) throws IOException {
> >          final List<String []> textTable = new ArrayList<>();
> >          for (int i = 0; i < v.getRowDimension(); i ++){
> > -            textTable.add(Arrays.stream(v.getRow(i)).mapToObj(Double :: toString).toArray(String[]::new));
> > +            textTable.add(Arrays.stream(v.getRow(i)).mapToObj(Double::toString).toArray(String[]::new));
>
> While you're at it, change v to a more matrixy name.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/gatk-protected/pull/609/files/0251a9fd487194dfafddd04bc67533d980891b2d#r70487732>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB5rL0IImx7BVAC0Dq1gMuvIjk9lzcXwks5qU9U2gaJpZM4JJrmA>
> .
>